app/models/spot.rb

```diff
@@ -0,0 +1,12 @@
+class Spot < ActiveRecord::Base
+  has_many :reviews
```

Comment:
A spot will have many reviews in the future, but since the review model doesn't exist yet, I would leave this line out